tools/obisindicators/robis.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="obis_data" name="OBIS occurences" version="@VERSION@" profile="20.01">
+    <description>retrieve data</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="topic"/>
+    <expand macro="requirements">
+        <requirement type="package" version="2.11.3">r-robis</requirement>
+    </expand>
+    <required_files>
+        <include type="literal" path="robis.r"/>
+    </required_files>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/robis.r'
+            '$species'
+            '$taxon'
+            '$lat_min'
+            '$lat_max'
+            '$long_min'
+            '$long_max'
+            '$output'
+        ]]>
+    </command>
+    <inputs>
+        <param name="species" type="text" label="Scientific name of the species" help="Genus species format, eg : Scomber scombrus"/>
+        <param name="taxon" type="text" label="Taxon ID"/>
```

Comment:
should we restrict this text filed a bit? Ony letters? only letter and numbers?

Reply:
Yes that's a good idea !

Reply:
Should I use a sanitizer like the following for species :
`            <sanitizer invalid_char="">
                <valid initial="string.ascii_letters">
                </valid>
            </sanitizer>`

Reply:
Mh I guess a validator is needed

Reply:
Ah thanks !!

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#id54